compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -251,6 +252,26 @@ static std::string translateModuleToISA(llvm::Module &module,
   return targetISA;
 }
 
+void checkRegisterSpilling(OpBuilder &builder, const std::string obj) {
+  uint16_t abi_version;
+  llvm::StringMap<llvm::offloading::amdgpu::AMDGPUKernelMetaData> info_map;
+
+  if (!llvm::offloading::amdgpu::getAMDGPUMetaDataFromImage(
+          llvm::MemoryBufferRef(obj, ""), info_map, abi_version)) {
+    for (auto &entry : info_map) {
```

Comment:
Please spell this out: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable